manifests/params.pp

```diff
@@ -60,9 +60,23 @@
     'st2auth',
     'st2debug',
   ]
-  $st2_client_packages = [
-    'python-st2client',
-  ]
+  case $::osfamily {
```

Comment:
I have no strong opinion... mostly a style question.

`in-selector` vs `case` here? What's your thought? After all these years, I still don't have a great definitive for answer here. Just guidelines.